graphql/client.js

```diff
@@ -0,0 +1,23 @@
+import { ApolloClient, InMemoryCache, HttpLink } from '@apollo/client'
+
+const link = new HttpLink({
+  uri: process.env.AEM_GRAPHQL_ENDPOINT,
+  headers: {
+    'User-Agent': 'secure-client-hub/0.0.1',
+  },
+})
+
+const client = new ApolloClient({
+  link: link,
+  cache: new InMemoryCache(),
+})
+
+export default async function (AEMQuery) {
+  return client
+    .query({
+      query: AEMQuery,
+    })
+    .catch((error) => {
+      console.log(error)
```

Comment:
should use our logger here

Reply:
This catch needs to be able to fail hard once we get a working example. I'll leave it for now

Reply:
Failing hard is fine; we should still log it properly.

Reply:
It is up to the parent caller to log so the full stack trace can be seen

Reply:
If I'm understanding what you're suggesting correctly, you want each call to AEM to handle it's errors individually so that more of the stack trace can be seen. Assuming the goal would be to know which call failed, why not just include the AEMQuery in the log, or pass an ID and put that in the log, which would reduce the duplication of having to log in multiple locations and also risk having each done differently.